packages/anoncreds/src/AnonCredsModule.ts

```diff
@@ -38,6 +38,20 @@ export class AnonCredsModule implements Module {
     dependencyManager.registerSingleton(AnonCredsLinkSecretRepository)
   }
 
+  public async initalize(agentContext: AgentContext) {
+    if (this.config.autoCreateLinkSecret === false) {
```

Comment:
Maybe could be the other way around? I.e. doing all the link secret stuff only if autoCreateLinkSecret is true?

Reply:
I can change it if you prefer but I think its better as is. Usually in an early return, its more common to handle edge cases, errors, or invalid input at the start of a function,  which lets the main function logic be less indented and easier to read.

LMK.